homeassistant/components/icloud/config_flow.py

```diff
@@ -229,18 +235,18 @@ async def async_step_trusted_device(self, user_input=None, errors=None):
             errors[CONF_TRUSTED_DEVICE] = "send_verification_code"
 
             return await self._show_trusted_device_form(
-                trusted_devices_for_form, user_input, errors
+                trusted_devices_for_form, errors
             )
 
         return await self.async_step_verification_code()
 
     async def _show_trusted_device_form(
-        self, trusted_devices, user_input=None, errors=None
-    ):
+        self, trusted_devices, errors: dict[str, str] | None = None
+    ) -> ConfigFlowResult:
         """Show the trusted_device form to the user."""
 
         return self.async_show_form(
-            step_id=CONF_TRUSTED_DEVICE,
+            step_id="trusted_device",
```

Comment:
I found it confusing to use the same constant for a config setting and for a step_id.
I have moved the `step_id` to be a regular string instead.